includes/utils/class-amp-image-dimension-extractor.php

```diff
@@ -32,11 +32,10 @@ public static function extract( $urls ) {
 		foreach ( $urls as $original_url ) {
 			$normalized_url = self::normalize_url( $original_url );
 			if ( false !== $normalized_url ) {
-				$url_map[ $normalized_url ] = $original_url;
+				$url_map[ $original_url ] = $normalized_url;
 				$normalized_urls[] = $normalized_url;
 			} else {
 				// This is not a URL we can extract dimensions from, so default to false.
-				$url_map[ $original_url ]           = $original_url;
```

Comment:
Why is this line being removed? It was recently added in #793.

Reply:
In the old version the line is unnecessary. The code after this line never uses url_map to get an url that cannot be normalized. Therefore, this line is unnecessary and can be removed.

In my version, it is also unnecessary and causes an error.